core/src/main/java/com/netflix/conductor/core/utils/EmailSender.java

```diff
@@ -0,0 +1,155 @@
+package com.netflix.conductor.core.utils;
```

Comment:
Please see comment in KafkaErrorEmailSender. Sending emails is not Conductor's responsibility and should be achieved by other means.

Reply:
Got rid of this class